tests/verify_collection.py

```diff
@@ -239,7 +240,19 @@ def get_lat_lon_var_names(dataset: xarray.Dataset, collection_variable_list: Lis
 
     # If that doesn't work, try using cf-xarray to infer lat/lon variable names
     try:
-        return dataset.cf.coordinates['latitude'][0], dataset.cf.coordinates['longitude'][0]
+        for lat in dataset.cf.coordinates['latitude']:
+            if lat in ['lat', 'latitude']:
```

Comment:
If we're only going to consider a specific list of strings as valid lat/lon coordinate names, let's extract that list into a constant so it's obvious and easier to add to later. Something like this up at the top of the file:

```
VALID_LATITUDE_VARIABLE_NAMES = ['lat', 'latitude']
VALID_LONGITUDE_VARIABLE_NAMES = ['lon', 'longitude']
```

Alternatively we could keep a list of invalid names/variable names we want to ignore and invert the logic

```
IGNORED_LATITUDE_VARIABLE_NAMES = ['fol_lat']
IGNORED_LONGITUDE_VARIABLE_NAMES = ['fol_lon']

...

if lat not in IGNORED_LATITUDE_VARIABLE_NAMES:
    ...

if lon not in IGNORED_LONGITUDE_VARIABLE_NAMES:
    ...
```